### DISCUSSION TITLE
[Feature] Multi-Select on "folder" view for external libraries (or search by external path)

### DISCUSSION THREAD

**Discussion by rholloway** [immich-app/immich#14118] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently there seems to be no way to select all photos from known external library path. For instance, if we have folders on an external library and want to create an immich album out of one of those folders, I can't find an easy way to do it. Two suggestions come to mind:

1. When using the folder view tab, enable multi-select (similar to other views in immich). This would allow us to go to a folder and then multi-select photos / select all and assign them to an album or tag or similar.
2. Alternatively, allow syntax or functionality to search photos by path. For instance, if the external library is in a (nested) folder called "Billys Birthday 2017" allow us to match photos within a path containing this string. Doing this would allow us to use the multi-select already present on search results page.

Right now, the closest workaround I've come across is to go to one of the photos from the folder view, then say "view in timeline", and then use multi select from there. However, this is really only useful for external library folders where they are from the same event or grouped together on the timeline, and is a bit of trouble to go and pick them out from the timeline when immich knows exactly which photos are already grouped together on the external folder.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9159/#6519

**Reply by nielstron** [reply]
now reopened in #14307
